website/docs/overview/fluree_basics.mdx

```diff
@@ -50,29 +53,43 @@ and `movie`. You can issue this transaction in your user interface. The syntax i
 explained in the [Transact](/docs/transact/basics) section. For now, we recommend
 you adding this basic schema and walking through the documentation section-by-section.
 
+<Tabs
+defaultValue="json"
```

Comment:
small thing, but i like indentation inside of an HTML tag for readability

Reply:
I indented then, but if you could double check that'd be great!